### PR TITLE
Perf counters descriptions

### PR DESCRIPTION
**Pull request by Ved-vampir**

### CODE REVIEW

File: src/client/Client.cc [hunk 1]
```diff
@@ -423,11 +423,11 @@ int Client::init()
 
   // logger
   PerfCountersBuilder plb(cct, "client", l_c_first, l_c_last);
-  plb.add_time_avg(l_c_reply, "reply");
-  plb.add_time_avg(l_c_lat, "lat");
-  plb.add_time_avg(l_c_wrlat, "wrlat");
-  plb.add_time_avg(l_c_owrlat, "owrlat");
-  plb.add_time_avg(l_c_ordlat, "ordlat");
+  plb.add_time_avg(l_c_reply, "reply", "Latency of receiving a reply on request");
+  plb.add_time_avg(l_c_lat, "lat", "Latency of processing a request");
```
Comment (gregsfortytwo) [inline src/client/Client.cc]: These two should probably indicate that they're about metadata ops.

File: src/client/Client.cc [hunk 2]
```diff
@@ -423,11 +423,11 @@ int Client::init()
...
+  plb.add_time_avg(l_c_wrlat, "wrlat", "Latency of write operation");
```
Comment (gregsfortytwo) [inline src/client/Client.cc]: And this one is a file data write.

File: src/client/Client.cc [hunk 3]
```diff
@@ -423,11 +423,11 @@ int Client::init()
...
+  plb.add_time_avg(l_c_owrlat, "owrlat", "Write to object latency");
+  plb.add_time_avg(l_c_ordlat, "ordlat", "Read from object latency");
```
Comment (gregsfortytwo) [inline src/client/Client.cc]: And these two, though present in the system, appear to be unused (except in the SyntheticClient). :(

File: src/client/Client.cc (hunk 1 above)
Comment (Ved-vampir) [inline src/client/Client.cc]: Will it be correct: "Latency of receiving a reply on metadata request"?

File: src/client/Client.cc (hunk 3 above)
Comment (Ved-vampir) [inline src/client/Client.cc]: Nonetheless SyntheticClient use it)

File: src/client/Client.cc (hunk 1 above)
Comment (gregsfortytwo) [inline src/client/Client.cc]: Yeah, looks good.

File: src/client/Client.cc [hunk 4]
```diff
@@ -423,11 +423,11 @@ int Client::init()
...
-  plb.add_time_avg(l_c_reply, "reply");
-  plb.add_time_avg(l_c_lat, "lat");
-  plb.add_time_avg(l_c_wrlat, "wrlat");
-  plb.add_time_avg(l_c_owrlat, "owrlat");
-  plb.add_time_avg(l_c_ordlat, "ordlat");
+  plb.add_time_avg(l_c_reply, "reply", "Latency of receiving a reply on metadata request");
+  plb.add_time_avg(l_c_lat, "lat", "Latency of processing a metadata request");
+  plb.add_time_avg(l_c_wrlat, "wrlat", "Latency of a file data write operation");
+  plb.add_time_avg(l_c_owrlat, "owrlat", "Write to object latency");
+  plb.add_time_avg(l_c_ordlat, "ordlat", "Read from object latency");
```
Comment (gregsfortytwo) [inline src/client/Client.cc]: What I mean about the SyntheticClient is that that's just a test tool (and one we don't use), so these last two are basically dead values. You can't expect to use them to get any meaningful data; I'm actually inclined to remove them but if not the description shouldn't imply that they'll give you info on OSD access times for your ceph-fuse mount. :(

Comment (liewegas) [inline src/client/Client.cc]: i vote to remove them!

Comment (Ved-vampir) [inline src/client/Client.cc]: Ok, removed.

**Comment by Ved-vampir** [comment]
@liewegas, @gregsfortytwo, need I change something else?

**Comment by gregsfortytwo** [comment]
I was just looking at the CephFS bits, hopefully @athanatos can comment on the other values.

**Comment by Ved-vampir** [comment]
Made separate commits, change comment and counter in throttle.

**Pull request merged**